spec/index.html

```diff
@@ -503,17 +505,23 @@ <h2>Strings in RDF</h2>
     <p>RDF uses Unicode [[Unicode]] as the fundamental representation for string values.
       Within this, and related specifications, the term <dfn id="dfn-rdf-string">string</dfn>,
       or <a data-lt="string">RDF string</a>,
-      `is used to describe an ordered` sequence of zero or more <a data-lt="code point" class="lint-ignore">Unicode code points</a> which are <a data-cite="i18n-glossary#dfn-scalar-value" class="lint-ignore">Unicode scalar values</a>. Unicode scalar values do not include the <a data-cite="i18n-glossary#dfn-surrogate"  class="lint-ignore">surrogate code points</a>.
+      `is used to describe an ordered` sequence of zero or more
```

Comment:
Why is this phrase styled as code?
```suggestion
      is used to describe an ordered sequence of zero or more
```

Reply:
I don't think the style was intended; I'm not sure where it was introduced.